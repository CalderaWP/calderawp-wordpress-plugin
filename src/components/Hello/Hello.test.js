import renderer from "react-test-renderer";
import React from "react";
import {
	HelloEdit,
	HelloDisplay
} from './index';

describe( "Display component", () => {
	it( "Matches snapshot", () => {
		expect(
			renderer.create(
				<HelloDisplay
					name={'Roy'}
				/>
			).toJSON()
		).toMatchSnapshot()
	});
});

describe( "Edit component", () => {
	it( "Matches snapshot", () => {
		expect(
			renderer.create(
				<HelloEdit
					name={'Roy'}
					label={'Change'}
					onChangeName={()=> {}}
				/>
			).toJSON()
		).toMatchSnapshot()
	});
});