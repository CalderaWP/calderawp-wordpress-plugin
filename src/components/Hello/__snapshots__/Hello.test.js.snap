// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component Matches snapshot 1`] = `
<div>
   
  Roy
</div>
`;

exports[`Edit component Matches snapshot 1`] = `
Array [
  <div
    className="components-base-control"
>
    <div
        className="components-base-control__field"
    >
        <input
            aria-describedby={undefined}
            className="components-text-control__input"
            id="inspector-text-control-0"
            onChange={[Function]}
            type="text"
            value="Roy"
        />
    </div>
</div>,
  <div
    className="components-base-control"
>
    <div
        className="components-base-control__field"
    >
        <input
            aria-describedby={undefined}
            className="components-text-control__input"
            id="inspector-text-control-1"
            onChange={[Function]}
            type="text"
            value={undefined}
        />
    </div>
</div>,
]
`;
